requirements.txt

```diff
@@ -1,3 +1,3 @@
 geoalchemy2==0.4.0
-numpy==1.12.0
+numpy==1.16
```

Comment:
Where is this used?

Reply:
I mean where is this setting in requirements.txt used?

Reply:
The requirements.txt is part of the old school (non-conda) python packaging process. Some people use it with pip to create an environment that can run the code.